server/CVCMIServer.cpp

```diff
@@ -615,9 +625,24 @@ int main(int argc, char** argv)
 		//and return non-zero status so client can detect error
 		throw;
 	}
+#ifdef VCMI_ANDROID
+	AndroidVMHelper envHelper;
+	envHelper.callStaticVoidMethod(AndroidVMHelper::NATIVE_METHODS_DEFAULT_CLASS, "killServer");
+#endif
 	delete VLC;
 	VLC = nullptr;
 	CResourceHandler::clear();
 
   return 0;
 }
+
+#ifdef VCMI_ANDROID
+
+void CVCMIServer::create()
+{
+	boost::this_thread::sleep(boost::posix_time::milliseconds(1000));

```

Comment:
I have no ideas how Android work at all and on desktop we can just start server process without any additional communication channel with it since we'll connect to it over network anyway. So interprocess only add complexity and cause issues on Windows.